tox.ini

```diff
@@ -11,7 +11,7 @@ deps =
     pyramid15: pyramid<=1.5.4
 
 commands =
-    coverage run --source=pyramid_swagger/ --omit=pyramid_swagger/__about__.py -m pytest --capture=no --strict {posargs}
+    coverage run --source=pyramid_swagger/ --omit=pyramid_swagger/__about__.py -m pytest --capture=no --strict tests/ {posargs}
```

Comment:
Pytest is always vigilant. Possibly a little too vigilant for my tastes :)

Instead of always including tests like this, lets make it the default value of posargs using http://tox.readthedocs.org/en/1.0/example/general.html#interactively-passing-positional-arguments.

That way you can always invoke like `tox -- tests/foobar_test.py` to avoid running the whole suite.